lib/build.gradle

```diff
@@ -23,6 +23,7 @@ kotlin {
 
 dependencies {
     implementation "com.nimbusds:nimbus-jose-jwt:${nimbusJoseJWTVersion}"
+    implementation "com.github.multiformats:java-multibase:${javaMultibase}"
```

Comment:
Please add 
`include(dependency("com.github.multiformats:java-multibase"))` into `dependencies` of `shadowJar`
(see https://github.com/sicpa-dlab/peer-did-jvm/blob/main/lib/build.gradle#L82)